CMakeLists.txt

```diff
@@ -840,7 +841,7 @@ install(TARGETS ${MXNET_INSTALL_TARGETS}
 
 install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/dlpack/include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/dmlc-core/include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
-if(USE_MKLDNN)
+if(INSTALL_MKLDNN)
```

Comment:
What's the problem with `USE_MKLDNN`?

Reply:
in cmake/BuildTVM.cmake I set `USE_MKLDNN` to OFF, does it affect this variable used later in root cmake? what is the best approach to scope an cmake option?

Reply:
I see. So both MXNet and TVM have this `USE_MKLDNN` flag but you don't want to enable it in TVM. Does the `<PARENT_SCOPE>` help here? https://cmake.org/cmake/help/latest/command/set.html#set-normal-variable

Reply:
actually I need something opposite of `PARENT_SCOPE`

```cmake
option(USE_MKLDNN ON)
if(USE_MKLDNN)
...
endif()
if(USE_TVM_OP)
  include(cmake/BuildTVM.cmake)   <- in BuildTVM.cmake I set(USE_MKLDNN OFF)
  ...
endif()
if(USE_MKLDNN) <- here it is OFF, while should be ON
...
endif()
```

Reply:
@yzhliu Yes, parent scope is affected in the current case. To fix this problem, you can make use of CMakes function scopes. Ie define a new function insede the BuildTVM.cmake, and do all variable changes inside that function.

You can take 
https://github.com/apache/incubator-mxnet/blob/3e638b4fc4de959c7afc614facc336a796347c9d/CMakeLists.txt#L407-L414
as a reference